src/components/AssetsClearFiltersButton/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@edx/paragon';
+
+import WrappedMessage from '../../utils/i18n/formattedMessageWrapper';
+import messages from './displayMessages';
+
+
+const AssetsClearFiltersButton = ({ clearFilters, courseDetails }) => (
```

Comment:
Yeah I'm not sure where to go with this. To me a search is something you use to filter, but we have also distinguished filters as something different than search in the code. I like calling it a "clear" because that's the action it performs, but what is it clearing? `AssetsClearFiltersAndSearchButton`, `AssetsClearControlsButton`, `AssetsClearQueryButton`...?